src/python/pants/backend/shell/target_types.py

```diff
@@ -430,13 +429,34 @@ class ShellCommandLogOutputField(BoolField):
 
 class ShellCommandWorkdirField(StringField):
     alias = "workdir"
-    default = None
+    default: ClassVar[Optional[str]] = "."
     help = softwrap(
-        "Sets the current working directory of the command, relative to the project root. If not "
-        "set, use the project root.\n\n"
+        "Sets the current working directory of the command. "
         "To specify the location of the `BUILD` file, use `.`. Values beginning with `.` are "
         "relative to the location of the `BUILD` file.\n\n"
-        "To specify the project/build root, use `/` or the empty string."
+        "To specify the build root, use `/` or the empty string.\n\n"
+        "Values that do not begin with `.` or `/` are relative to the build root."
```

Comment:
aren't paths that begin with `/` going from the build root?

Reply:
Yes. So `/boop` and `boop` are the same thing. Is your issue that I didn't spell out the `/boop` form in the documentation?

Reply:
Yes, sorry no, my issue is with:

> Values that do not begin with `.` or `/` are relative to the build root.

which would imply that paths beginning with `/` are not relative to build root.

Reply:
@kaos does https://github.com/pantsbuild/pants/pull/18157/commits/b246e604d92976e26155956d08066f835d23d05b look better?

Reply:
Yep, that's better. Feeling some repetitiveness going on in the bullet list, but prefer being overly explicit rather than vague any day.